### DISCUSSION TITLE
谷歌商店可以浏览。无法下载

### DISCUSSION THREAD

**Discussion by Leno709394** [kkkgo/PaoPaoDNS#35] [问答和帮助]
同一个节点。手机v2ray app 代理。正常浏览下载。
挂到软路用paopao。 就只能浏览无法下载
添加这两到强制走代理。连浏览都不行了。 两个加到直连又回到能浏览。不能下载
![image](https://github.com/kkkgo/PaoPaoDNS/assets/104353759/0f913277-adeb-4ce6-866d-158a50b1c3f7)

https://www.v2ex.com/t/696735?p=2  参考了此贴153楼方法。不行。不知道那里出问题

**Comment by kkkgo** [comment]
你说的代理是跟ppgw一起使用吗？
force_nocn_list是境外原始解析结果，如果配合ppgw代理的话应该是force_forward_list。
可以跑一下`debug.sh`看日志吗

**Comment by kkkgo** [comment]
另外，建议开启USE_MARK_DATA功能，你圈起来的域名第一条是有覆盖到的，第二条我查了应该是无效域名。
![image](https://github.com/kkkgo/PaoPaoDNS/assets/4530897/3f21d599-40a8-4dd6-8f12-884614b85088)

**Comment by kkkgo** [comment]
参考了你发的贴子的99楼，域名应该是`r2---sn-i3belnlz.xn--ngstr-lra8j.com`，但无论境内或者境外解析都是谷歌香港地址，并且数据库也在。还是需要你补充更多信息。一种有可能的情况就是跟99楼、101楼说的一样，是节点问题，但如果客户端是用IPOnDemand，那么可能不是走远程解析，而ppgw为了更快的速度，是所有解析都是远程解析的。

**Reply by Leno709394** [reply]
解决了，谢谢。说起来也是犯傻。force_forward_list一开始填这里的，不生效才填了另一个去。。听你这么说。我又填回去force_forward_list 然后重启dns服务器。。结果可以了。估计是刚好卡bug。 因为之前添加是不用重启就生效的

**Reply by kkkgo** [reply]
但是你填的这两个域名都在USE_MARK_DATA的数据库里，你直接开启USE_MARK_DATA功能就可以了。

**Reply by kkkgo** [reply]
`USE_MARK_DATA`功能 参考更新日志：https://github.com/kkkgo/PaoPaoDNS/discussions/28

**Reply by Leno709394** [reply]
没发现后期更新了这功能。的确直接打开USE_MARK_DATA 也可以了。不用自己添加规则了

**Comment by kkkgo** [comment]
> 解决了，谢谢。说起来也是犯傻。force_forward_list一开始填这里的，不生效才填了另一个去。。听你这么说。我又填回去force_forward_list 然后重启dns服务器。。结果可以了。估计是刚好卡bug。 因为之前添加是不用重启就生效的

修改的确是不用重启就生效的，实时监控修改重载（只要权限没问题的话，有些修改的用户和运行用户不一样，监控不出来）。

**Comment by kkkgo** [comment]
https://github.com/kkkgo/PaoPaoDNS/discussions/45
